app/models/ability.rb

```diff
@@ -1,20 +1,16 @@
 class Ability
   include CanCan::Ability
-
   def initialize(user)
-    if user.isadmin?
-      can :manage, :all
+    can :read, Post
+    can :read, Comment
+
+    return unless user.present?
+
+    can :manage, Post
+    can :manage, Comment
```

Comment:
- [ ] Great job now it's working but as a regular user, I was able to delete other's posts please make sure that not happening so that only the author of the post can delete it

Reply:
Thank you very much for the review @youmari  , I have made the requested changes for this